assets/elm/Data/Board.elm

```diff
@@ -128,8 +155,13 @@ nextSnakeAndApples time snake apples =
 
 
 score : Board -> Int
-score { snake } =
-    List.length snake.body
+score { currentPlayerId, snakes } =
+    case Dict.get currentPlayerId snakes of
+        Nothing ->
+            0
```

Comment:
Actually, if refactoring, I'd probably do this:

```
-- Snake module

bodyLength : Snake -> Int
bodyLength = List.length << .body


-- other module

score : Board -> Int
score { currentPlayerId, snakes } =
    let
        maybeCurrentPlayerSnake = getSnakeForPlayer currentPlayerId snakes
    in
        maybe 0 bodyLength maybeCurrentPlayerSnake


getSnakeForPlayer : PlayerId -> Dict PlayerId Snake -> Maybe Snake
getSnakeForPlayer playerId snakes = 
    Dict.get currentPlayerId snakes
```

but, yeah, like I said, I hadn't gotten that far yet :)

I got a bit more done on the elixir side last night, but I haven't pushed it up yet. Want to get the elm stuff working with the elixir stuff first.